orangecontrib/network/widgets/OWNxExplorer.py

```diff
@@ -268,6 +272,7 @@ def mark_more_than_average_neighbour():
 
         self.mark_criteria = [
             ("(Select criteria for marking)", None, lambda: np.zeros((0,))),
+            ("Mark node labelled as", text_line(), mark_label_equals),
```

Comment:
"Labelled" is BE, "labeled" is AE. As Orange is written in AE, I suggest changing it to "labeled".